modules/transport-netty4/src/test/java/org/elasticsearch/http/netty4/Netty4HttpRequestSizeLimitIT.java

```diff
@@ -109,21 +108,19 @@ public void testLimitsInFlightRequests() throws Exception {
     public void testDoesNotLimitExcludedRequests() throws Exception {
         ensureGreen();
 
-        @SuppressWarnings("unchecked")
-        Tuple<String, CharSequence>[] requestUris = new Tuple[1500];
-        for (int i = 0; i < requestUris.length; i++) {
-            requestUris[i] = Tuple.tuple("/_cluster/settings",
-                "{ \"transient\": {\"search.default_search_timeout\": \"40s\" } }");
+        ArrayList<Tuple<String, CharSequence>> requestUris = new ArrayList<Tuple<String, CharSequence>>();
```

Comment:
I would abbreviate this to:

```suggestion
        List<Tuple<String, CharSequence>> requestUris = new ArrayList<>();
```

Reply:
Thank you for your suggestion. I just pushed a new commit.